source/adapters/cuda/platform.cpp

```diff
@@ -168,5 +168,10 @@ UR_APIEXPORT ur_result_t UR_APICALL urPlatformGetBackendOption(
     *ppPlatformOption = "";
     return UR_RESULT_SUCCESS;
   }
+  if (pFrontendOption == "-foffload-fp32-prec-div" ||
+      pFrontendOption == "-foffload-fp32-prec-sqrt") {
+    *ppPlatformOption = "";
```

Comment:
TBH I'm not sure if it's needed.

Reply:
It's fine for JIT, but you are right should we want to support it there, it would require more work.

Reply:
So it's fine to leave it as is?

Reply:
Yeap, I would say so.